packages/core/src/RenderingEngine/helpers/createVTKImageDataFromImage.ts

```diff
@@ -0,0 +1,24 @@
+import { IImage } from '../../types';
+import createVTKImageData from './createVTKImageData';
+import { getSegmentationImageFromImageId } from './getDerivedImage';
+import { getMetadataFromImage } from './getMetadataFromImage';
+
+export default function createVTKImageDataFromImage(image: IImage) {
+  const { origin, direction, dimensions, spacing } =
+    getMetadataFromImage(image);
+
+  const imagedata = createVTKImageData({
+    origin,
+    direction,
+    dimensions,
+    spacing,
+    numComps: 1,
+    pixelArray: image.getPixelData(),
+  });
+  return imagedata;
+}
+
+export function createVTKImageDataFromImageId(imageId) {
+  const image = getSegmentationImageFromImageId(imageId);
```

Comment:
Should not exist, the creation of segmentation image from image Id should happen inside the tools